inc/woocommerce/storefront-woocommerce-template-functions.php

```diff
@@ -214,18 +201,6 @@ function storefront_loop_columns() {
 	}
 }
 
-if ( ! function_exists( 'storefront_product_columns_wrapper_close' ) ) {
-	/**
-	 * Product columns wrapper close
-	 *
-	 * @since   2.2.0
-	 * @return  void
-	 */
-	function storefront_product_columns_wrapper_close() {
-		echo '</div>';
-	}
-}
-
```

Comment:
Could child themes or other customizations be using these functions (`storefront_product_columns_wrapper` and `storefront_product_columns_wrapper_close`)? If they're available there may be stores that are using them. 

If we intend to remove them we'll need to deprecate and document any changes so stores can adapt their customisations.

(I'm assuming there's low chance of stores using these, but we need to take care just in case.)